js/components/utils.js

```diff
@@ -106,19 +105,19 @@ var Utils = {
       if (useNegativeDvrOffset) {
         // We don't show current time unless there is a time shift when using
         // negative DVR offset
-        currentTime = isLiveNow ? null : Utils.formatSeconds(timeShift);
+        currentTime = isLiveNow ? '' : this.formatSeconds(timeShift);
       } else {
         // When not using negative DVR offset, DVR progress is shown in the usual
         // "current time of total time" format, with total time set to the size of DVR window
-        currentTime = isLiveNow ? totalTime : Utils.formatSeconds(Math.ceil(duration + timeShift));
+        currentTime = isLiveNow ? totalTime : this.formatSeconds(Math.ceil(duration + timeShift));
       }
     } else {
       currentTime = currentPlayheadTime;
     }
     // Total time is not displayed when using negative DVR offset, only the
     // timeshift is shown
     if (useNegativeDvrOffset) {
-      totalTime = isLiveStream ? null : totalTime;
+      totalTime = isLiveStream ? '' : totalTime;
```

Comment:
There is no good way of "fixing" the output when the input parameters are incorrect, so instead of handling all of the edge cases I'm now returning empty strings instead of `null`'s. This way at least the output values can be consistently treated as strings.